src/components/login/sagas.js

```diff
@@ -0,0 +1,21 @@
+import { takeLatest } from 'redux-saga/effects';
+import { AUTHENTICATE_USER } from './actionTypes';
+
+/**
+ * Watches for the {@link actionTypes.authenticateUser authenticateUser} action.
+ * Gets the requested data from the server.
+ *
+ * @return {void}
+ */
```

Comment:
what is the line 4-9 comment for?

Reply:
it is for the authenticator in the version i am yet to push. this pull request was for setting up the sagas and redux